caluma/settings.py

```diff
@@ -125,3 +126,12 @@ def parse_admins(admins):  # pragma: no cover
 OIDC_VALIDATE_CLAIMS_OPTIONS = env.dict(
     "OIDC_VALIDATE_CLAIMS_OPTIONS", cast={"value": bool}, default=None
 )
+
+# Extensions
+
+VISIBILITY_CLASSES = [
+    import_string(cls)()
```

Comment:
In my understanding, when you call `import_string(cls)()` like this, you'll get an instance, not a class. So either the setting's name is wrong or you should only fetch the classes

Reply:
fair point. changed it so there are actual classes here.